### ISSUE TITLE
About modal

### ISSUE THREAD

**Issue by thegurf** [iluaepidi/rating-tool#9]
When open, the about modal dialog does not trap keyboard focus, making it possible for screenreader users to wrap focus to the top of the page and interact content that is behind the open modal dialog box overlay. 

When the overlay is open, keyboard focus should be restricted to the close control and the content within the overlay.

**Comment by p-sl** [comment]
Library used for modal content restricts by default the navigation within modal content. It shuld not be possible navigate outside modal content.

**Comment by thegurf** [comment]
haven't been able to reproduce this one again, so closing
